gate-proxy/src/main/kotlin/com/netflix/spinnaker/gate/controllers/ApiExtensionController.kt

```diff
@@ -48,7 +48,8 @@ class ApiExtensionController @Autowired constructor(private val apiExtensions: L
     val duplicateApiExtensionIds = apiExtensions
       .groupBy { it.id().toLowerCase() }
       .filter { it.value.size > 1 }
-      .flatMap { it.value }
```

Comment:
there are some incompatible api changes to Collections API . Need to modify this to get it compiled.